client/blocks/location-search/README.md

```diff
@@ -0,0 +1,32 @@
+LocationSearch
+===
+
+A search component for searching locations via the Google Places API.
+
+## Usage
+
+```jsx
+import LocationSearch from 'blocks/location-search';
+
+export default function AwesomeLocationSearch() {
+	return (
+		<LocationSearch />
+	);
+}
+```
+
+### Props
+
+Props are displayed as a table with Name, Type, Default, and Description as headings.
+
+**Required props are marked with `*`.**
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`onPredictionClick` | `func` | `undefined` | Click handler for a search suggestion
+`predictionHref` | `string` | `undefined` | If provided, renders `a` instead of `button` for a suggestion
```

Comment:
Why are we using the term _prediction_ here? It's not clear to me what a prediction is in this context. What about something more general like _suggestion_ (as mentioned in the descriptions), or even _result_?

Reply:
I see now why you used this term now that I've reviewed the rest of the code. I would still recommend renaming it as _prediction_ is more an implementation detail inherited from the Google Places API, and anyone using this new component shouldn't be concerned about this.

Reply:
Prediction is the term the Google API uses, so we might as well keep using it so we'll keep using the same terms.